src/Providers/Gemini/Handlers/Structured.php

```diff
@@ -0,0 +1,101 @@
+<?php
```

Comment:
Again i guess this is correct but can you please keep the methods of this structured output as consistent across other provider so its easier to update or refactor any thing related to provider. for eg `src/Providers/Anthropic/Handlers/Structured.php` 
Again this optional.

@sixlive what do you think.